dracut/30ignition/ignition-reboot.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Ignition (reboot)
+DefaultDependencies=false
+Before=ignition-complete.target
+
+# Make sure ExecStart= runs before we switch root
+Before=initrd-switch-root.target
+
+# Make sure root filesystem is mounted
```

Comment:
nit: `ignition-mount` does not actually mount the rootfs; it mounts all other filesystems defined by the Ignition config.

Reply:
Thanks for explaining!
One followup question is that, is it `initrd-root-fs.target` that mounts the rootfs?

Reply:
Check out `bootup(7)` and `systemd.special(7)`. Those are super helpful for understanding the boot process!